llmebench/datasets/SemEval23T3Propaganda.py

```diff
@@ -1,19 +1,18 @@
 import json
-import os
 from pathlib import Path
 
 from llmebench.datasets.dataset_base import DatasetBase
 
 
-class PropagandaSemEval23Dataset(DatasetBase):
+class SemEval23T3PropagandaDataset(DatasetBase):
     def __init__(self, techniques_path=None, **kwargs):
         # Get the path to the file listing the target techniques
         self.techniques_path = Path(techniques_path) if techniques_path else None
-        super(PropagandaSemEval23Dataset, self).__init__(**kwargs)
+        super(SemEval23T3PropagandaDataset, self).__init__(**kwargs)
 
     def metadata():
         return {
-            "language": "multilingual",
+            "language": ["en", "es", "fr", "ge", "gr", "it", "ka", "po", "ru"],
```

Comment:
"ge" -> "de" (german), "gr" -> "el" (greek), "po" -> "pl" (polish) are the correct ISO language codes, I've made the changes in a subsequent comment but just wanted to let you know incase I misread the intended languages